### ISSUE TITLE
Microsoft really want people to use this?

### ISSUE THREAD

**Issue by Ricardoaraujo00** [aspnet/AspNetCore.Docs#16279]
Sorry Shayne Boyer but there as parts of this explanation that are hard to follow, for example, where to place this?

```
app.UseForwardedHeaders(new ForwardedHeadersOptions
{
    ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto
});

app.UseAuthentication();
```
I don't know where is the Startup.Configure. I just have appsettings.json and lauchSettings.json. I'm lost right in the beginning, wich, despite being a beginner, is very bad.

I think if Microsoft wants us, developers, to use their technology, "she"(the company) could make things a little easier for us, and everyone, even you employees. These explanations are very hard to follow and replicate! You guys could make video explanations, making things work in those videos, and not only present these texts as guidelines. Come'on, it's 2019.. We need video(at least as side note). Text is so last century.. Come'on, please. Make videos.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cfd9a1f0-c1b6-4443-0700-4c5beafeed03
* Version Independent ID: c22ca7a0-d1d4-200d-fc36-a9798d350c6e
* Content: [Host ASP.NET Core on Linux with Apache](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-apache?view=aspnetcore-3.1)
* Content Source: [aspnetcore/host-and-deploy/linux-apache.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/linux-apache.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **shboyer**

**Comment by guardrex** [comment]
The trouble with videos is that they become dated fairly fast. Then, the entire video has to be replaced because it isn't easy to cut in new or changed content. Text is going to be with us for a long time.

> I'm lost right in the beginning, wich, despite being a beginner, is very bad.

*Don't try to consume this topic directly.* This topic builds upon fundamental concepts in other topics. Start with the topics in the *Fundamentals* node. You probably should even access the [.NET Core doc set](https://docs.microsoft.com/dotnet/core/index) even before you do that ... I mean just skimming those and reading the most important conceptual and tooling docs.

WRT something more like a tutorial for Linux, we received exactly that feedback for IIS ... that the uber IIS topic was complex and overloaded for a simple *first experience*. We agreed, so we put up the [Publish an ASP.NET Core app to IIS](https://docs.microsoft.com/aspnet/core/tutorials/publish-to-iis) tutorial. That was worked on ...

* https://github.com/aspnet/AspNetCore.Docs/issues/13309 - the issue
* https://github.com/aspnet/AspNetCore.Docs/pull/13593 - the PR

Because there might be a similar interest for the two Linux scenarios (Nginx and Apache), let's ask @danroth27 ... Are tutorials for Linux-Nginx Linux-Apache best for us to do, or should this be something left to the community?

(Stand-by @Ricardoaraujo00 for an answer tho ... the ⛄️ *holidays* 🎁 are here ... many are out and will be OOF until after New Years.)

**Comment by Rick-Anderson** [comment]
> I don't know where is the `Startup.Configure`

This is not a beginners document. If you don't know about `Startup.Configure` you should start with https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start, https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api, or similar.

**Comment by Ricardoaraujo00** [comment]
Thank you @Rick-Anderson 
Despite had developed a schedule management project in school in aspnet core and made some api's i'm still not very familiar with somethings. I'll look that into more deep.

About the videos you may have a point but thanks to this great soul(https://www.youtube.com/watch?v=7DwfRtBbvv8) i got to learnd what would be most certainly impossible following microsoft documentation..
